src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="form-group">
+    <label>{{'flow-builder.output-branching' | trans}}</label><br/>
+    <div class="btn-group">
+      <button @click="setBranching('segregated')"
+              class="btn btn-sm"
+              :class="{
+                active: isBranchingTypeSegregated,
+                'btn-primary': isBranchingTypeSegregated,
+                'btn-secondary': !isBranchingTypeSegregated,
+              }"
+              data-toggle="tooltip"
+              data-placement="bottom"
+              :title="'flow-builder.separate-output-for-each-choice' | trans">
+        <i class="v5icon-branching-on v5icon-2x"></i>
+      </button>
+      <button @click="setBranching('unified')"
+              class="btn btn-sm"
+              :class="{
+                active: isBranchingTypeUnified,
+                'btn-primary': isBranchingTypeUnified,
+                'btn-secondary': !isBranchingTypeUnified,
+              }"
+              data-toggle="tooltip"
+              data-placement="bottom"
+              :title="'flow-builder.one-output-for-all-choices' | trans">
+        <i class="v5icon-branching-off v5icon-2x"></i>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+  import { get, isNil } from 'lodash'
+  import { Component, Prop} from 'vue-property-decorator'
+  import { IBlock } from '@floip/flow-runner'
+  import { mixins } from 'vue-class-component'
+  import Lang from '@/lib/filters/lang'
+  import { namespace } from 'vuex-class'
+
+  const flowVuexNamespace = namespace('flow')
+
+  const BRANCHING_TYPE_UNIFIED = 'unified'
+  const BRANCHING_TYPE_SEGREGATED = 'segregated'
+
+  @Component({})
+  export default class BlockOutputBranchingConfig extends mixins(Lang){
+    @Prop() readonly block!: IBlock
+
+    branchingType: string = BRANCHING_TYPE_SEGREGATED
+
+    get isBranchingTypeSegregated() {
+      return this.branchingType === BRANCHING_TYPE_SEGREGATED
+    }
+
+    get isBranchingTypeUnified() {
+      return this.branchingType === BRANCHING_TYPE_UNIFIED
+    }
+
+    mounted() {
+      // @ts-ignore TODO: remove this once IBlock has vendor_metadata key, and confirm this path because FLOIP spec doesn't provide a definition for the output branching persistence
+      let value = get(this.block.vendor_metadata, 'branchingType')
+      if (isNil(value) && this.block) {
+        this.block_updateVendorMetadataByPath({ blockId: this.block.uuid, path: 'branchingType', value: this.branchingType })
```

Comment:
@bzabos , this is another proposal I made. In fact, we don't have a prop to save the `branchingType` in the floip spec. In voto5, we save it inside `customData.branching`. I didn't hesitate to put it inside vendor_metadata because we already add something like block UI position there.

Any concerns ?

Reply:
After call: we will leave it under vendor_metadata, & prefix it with `io_viamo`